_posts/2020-10-10-the-importance-of-time-tracking.md

```diff
@@ -0,0 +1,47 @@
+We've had some trouble over the years with our time tracking that led us to work more time than what we finally billed for. We would like to share some of the experience we have collected over the years and the tools that we currently use.
+
+<!--more-->
+
+## Implications of poor time tracking
+Poor time tracking can cause direct and indirect damage to this kind of companies.
+
+### Direct
+- Impact in revenue: Less hours tracked for client projects means less hours billed at the end of the billing period for the same amount of work. Over delivering is a great practice, but only if it is done on purpose. If not, it may go unnoticed.
+
+- Miscalculation of internal project costs: This cost is proportional to the time invested on this projects, so time tracking is vital on this subject.
+
+### Indirect
+- Time estimations for billable projects may turn unreliable and hourly rates can become affected as a result.
+
+- People tending to track less time than what they actually worked reflects directly in the company's efficiency, making it look higher than what it actually is.
+
+- Internal processes, like calls, may be taking more time than necessary. Properly tracking the time spent on these calls is the best way of troubleshooting internal efficiency issues.
+
+## Improving Time Tracking Skills
+Some people struggle to keep up with their time tracking. From our experience we've learnt that sometimes it's a matter of coming across a system that is suitable for you.
+
+### Our Process
+For logging our time into the projects we work on, creating invoices for our clients and tracking their payment, we like to use [nokotime.com](nokotime.com).
+
+You can use the main website to submit your entries, but we found this impractical since you need to recall the amount of time you spent on which task, and then submit your entry.
+To avoid this, you can take advantage of their timer. It's a downloadable app for ios/mac and a Google Chrome extension. This timer lets you track the time you spend daily on your projects and add descriptions to your entries. When you finish with a task you can submit your time entry and start a new one on the go.
+
+Getting used to using a timer like this prevents you of having a hard time to remember of all the tasks you worked on before you submit your entry.
+
+It is also a great idea to follow the [Pomodoro Technique](https://todoist.com/productivity-methods/pomodoro-technique) to increase productivity, and it's totally adaptable to any timer you may be using for your time tracking.
+
+Another app some of us like to use is [WakaTime](https://wakatime.com/). Waka is thought mainly for developers, since it integrates with several [different editors](https://wakatime.com/plugins) through their plug-ins. It automatically tracks the project you're working on and the time you spend working on it.
```

Comment:
```suggestion
Another app some of us like to use is [WakaTime](https://wakatime.com/). Waka is used mainly for developers, since it integrates with several [different editors](https://wakatime.com/plugins) through their plug-ins. It automatically tracks the project you're working on and the time you spend working on it.
```